app/models/database.py

```diff
@@ -1,26 +1,60 @@
 from typing import Optional
 from boto3.session import Session
-from app.service.url_shortener import Link
+from app.models.link import Link
 import os
 
 # this file will contain connection to the database, and database logic
 
 
 class LinkDatabase:
```

Comment:
You'll want to create a global instance of LinkDatabase and access it rather than create a new one each time you have a function call.

Reply:
Noted, 
i moved the db outside the function, to make it global